components/Templates/includes/auto-template/Pods_Templates_Auto_Template_Front_End.php

```diff
@@ -192,7 +193,7 @@ public function auto_pods() {
 				$options  = pods_v( 'options', $pod_data, array() );
 
 				// if auto template is enabled add info about Pod to array.
-				if ( 1 === (int) pods_v( 'pfat_enable', $options ) ) {
+				if ( pods_v( 'pfat_enable', $options, false ) ) {
```

Comment:
This is not a boolean and could come through as a string here. Better to enforce on this.

```suggestion
				if ( 1 === (int) pods_v( 'pfat_enable', $options, 0 ) ) {
```

Reply:
The field is set to a boolean type so why would it return anythinig else than a 1 or 0?
Anyway, fine with me. I think it would be nice to have the 2.8 PodsObject to parse the options correctly so things like this aren't needed anymore :)